src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -66,20 +86,109 @@ export class GithubAppsLogsComponent implements OnInit {
       : ['repository', 'entryName', 'deliveryId', 'reference', 'success', 'type'];
     this.displayedColumns = ['eventDate', 'githubUsername', ...this.columnsToDisplay];
     this.loading = true;
-    this.dataSource.sort = this.sort;
-    this.dataSource.paginator = this.paginator;
-    const lambdaEvents = this.matDialogData.userId
-      ? this.lambdaEventsService.getUserLambdaEvents(this.matDialogData.userId)
-      : this.lambdaEventsService.getLambdaEventsByOrganization(this.matDialogData.organization);
+  }
+
+  ngAfterViewInit() {
+    this.loadAppsLogs();
+  }
+
+  loadAppsLogs() {
+    setTimeout(() => {
```

Comment:
Do you need this, and if so why? If you do need it, a code comment would be good.

Reply:
I thought I needed it because I was using https://github.com/dockstore/dockstore-ui2/blob/62fa718933b3eb852602cb10defab94c503faeb7/src/app/shared/tool-lister.ts#L68 to help create this ticket but when I tested it without it, it looks to work the same!